build.sbt

```diff
@@ -12,6 +12,8 @@ scalacOptions := Seq(
   "-unchecked"
 )
 
+ensimeIgnoreScalaMismatch in LocalProject("fpscala") := true
```

Comment:
if this is added, and I build this project on a machine without ensime, it will fail won't it?